libcodechecker/server/server.py

```diff
@@ -412,8 +414,9 @@ def do_POST(self):
                 else:
                     # Send a custom, but valid Thrift error message to the
                     # client requesting this action.
-                    error_msg = "API version v{0} not supported by server." \
-                                .format(api_ver)
+                    error_msg = "Incompatible client/server API." \
```

Comment:
Why remove the version the client is trying to connect with from the error message? I think this makes uncovering the error harder.

Reply:
It is removed from here but it is printed in `client.py`